metacontrol_sim.rosinstall

```diff
@@ -24,3 +24,18 @@
 - git:
     local-name: metacontrol_move_base_configurations
     uri: https://github.com/rosin-project/metacontrol_move_base_configurations
+- git:
+    local-name: mros1_reasoner
+    uri: https://github.com/tud-cor/mc_mros_reasoner.git
+    version: split_mros1_reasoner
+- git:
+    local-name: metacontrol_msgs
+    uri: https://github.com/rosin-project/metacontrol_msgs
+- git:
+    local-name: mc_mdl_tomasys
+    uri: https://github.com/tud-cor/mc_mdl_tomasys.git
+    version: v0.1
+- git:
+    local-name: catkin_virtualenv
+    uri: https://github.com/locusrobotics/catkin_virtualenv.git
+    version: 0.6.1
```

Comment:
Not sure where this was ticketed again, but `catkin_virtualenv` was synced to Melodic: [status_page/ros_melodic_default.html?q=catkin_v](http://repositories.ros.org/status_page/ros_melodic_default.html?q=catkin_v).

Reply:
> Not sure where this was ticketed again

It's on this issue  https://github.com/tud-cor/mc_mros_reasoner/issues/88

I'll test it, and if it works and change it

Reply:
OK, I've removed it and also fixed the mros_reasoner version in https://github.com/rosin-project/metacontrol_sim/commit/29f30f76be693aa648d858ae5e6be37d1bc69990